src/jolt/util/feo_calib.py

```diff
@@ -0,0 +1,296 @@
+from jolt.util.measurement import (
+                                    repeated_get,
+                                    await_set,
+                                    await_set_stabilized,
+                                  )
+
+from jolt.driver.joltcb import Channel, JOLTComputerBoard
+from jolt.util.array import arange
+from typing import List, Tuple, Optional
+import logging
+import statistics
+from datetime import datetime, timedelta
+import csv
+import time
+from appdirs import AppDirs
+from pathlib import Path
+from threading import Event
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)

```

Comment:
Typically, you don't want to change it in a module that is loaded. It should only be done in the "main" module.